src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -1230,12 +1230,6 @@ TEST(DirectTaskTransportTest, TestSchedulingKeys) {
                     BuildTaskSpec(resources1, descriptor1),
                     BuildTaskSpec(resources2, descriptor1));
 
-  // Tasks with different function descriptors do not request different worker leases.
```

Comment:
Why remove this?

Reply:
Because we've added function descriptors to the scheduling class so we _do_ want them to request different leases now?